### ISSUE TITLE
Highlighting doesn't like empty child classes.

### ISSUE THREAD

**Issue by tadeassoucek** [ALANVF/vscode-pascal-magic#1]
Suppose this code:

~~~ pascal
unit abc;

interface

  type
    abc = class;

implementation

  function Something: Boolean;
  begin
    Something := not true
  end;

end.
~~~

Highlighting this with vscode-pascal-magic works as intended:

![code1](https://user-images.githubusercontent.com/39809382/99388560-124bc500-28d6-11eb-9f06-9fed4610fdbc.png)

But when you try to make `abc` inherit from another class like this:

~~~ pascal
unit abc;

interface

  type
    abc = class(Exception);

implementation

  function Something: Boolean;
  begin
    Something := not true
  end;

end.
~~~

the syntax highlighting stops working properly:

![code2](https://user-images.githubusercontent.com/39809382/99388579-18da3c80-28d6-11eb-94f9-e22f5455a913.png)

You can circumvent this by typing `class(Exception) end;`, but it'd still be great to be able to use the `class(Exception);` syntax in code.

**Comment by ALANVF** [comment]
Hmm, I thought that the syntax for empty derived classes was `abc = class of Exception;`, but yeah that seems like a pretty simple fix

**Comment by ALANVF** [comment]
Ok, should be fixed as of version 0.1.6 :+1:
